exchange/bitswap/workers.go

```diff
@@ -9,16 +9,20 @@ import (
 	u "github.com/ipfs/go-ipfs/util"
 )
 
+var TaskWorkerCount = 4
```

Comment:
i dont think 4 is enough? 

`4 * 256k = 1MB`. i think the workers have to wait for the other side so a lot of time may be spent waiting for requests. this will be particularly bad in LAN settings where your 4 workers got stuck sending to 4 remote peers, but everyone else locally who you could be sending to is stuck waiting.

What's the cost of these? just a goroutine queue? if so, i'd crank it up to 16/32.